e2e/deployers/deployers.go

```diff
@@ -0,0 +1,44 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+package deployers
+
+import (
+	"fmt"
+
+	"github.com/ramendr/ramen/e2e/types"
+)
+
+var registry map[string]types.Deployer
+
+func init() {
```

Comment:
I think we should restrict the use of init() for truly init scenarios. Would have preferred a getOrBuildRegistry() function to maintain the registry which can be called wherever registry is required.